frontend/src/Components/NotFound.js

```diff
@@ -25,7 +26,7 @@ NotFound.propTypes = {
 };
 
 NotFound.defaultProps = {
-  message: 'You must be lost, nothing to see here.'
+  message: 'DefaultNotFoundMessage'
```

Comment:
translate needs to be called directly on `DefaultNotFoundMessage`, if it doesn't work within `defaultProps` then message shouuld not be required and it can be done when destructuring the props.

```
function NotFound(props) {
  const { message = translate('DefaultNotFoundMessage') } = props
...
```

Reply:
think i've got what you mean. does this still work if an alternative message is passed in to NotFound (eg https://github.com/Sonarr/Sonarr/blob/c57ceac4debf7419be84096f997ba7b75c906586/frontend/src/Series/Details/SeriesDetailsPageConnector.js#L55-L59)

Reply:
No, but it should be translated here instead.

```
return ( 
   <NotFound 
     message={translate('SeriesCannotBeFound')} 
   /> 
 ); 
```

That way we don't have any mysteriously missing translations (hopefully).

Reply:
> No, but it should be translated here instead.

that one I will get to when I translate the fronted/series pages

so for now, this one is resolved I think?

Reply:
Yeah, all good now